src/core/load-fragment.c

```diff
@@ -844,6 +844,11 @@ int config_parse_exec_input(
         assert(line);
         assert(rvalue);
 
+        if (isempty(rvalue)) {
+                c->std_input = EXEC_INPUT_NULL;
+                return 0;
+        }
```

Comment:
i have the suspicion eventually, one day we should rework these two enums btw, and make them default to _EXEC_INPUT_INVALID or so, so that we can distuingish the NULL setting from an "unset" setting. but then again, this PR doesn't and shouldn't cover that...